catalog/app/components/Preview/renderers/Perspective.js

```diff
@@ -0,0 +1,102 @@
+import cx from 'classnames'
```

Comment:
why is this file JS and not TS?

Reply:
I started with perspective@1.0.3, and now it's 1.2.0 :)
Type definitions work now, but earlier they were buggy